docs/source/cli_help/github.rst

```diff
@@ -0,0 +1,42 @@
+.. role:: bash(code)
+
+.. _github:
+
+github
+------
+Usage:
+
+.. code-block:: shell
+
+    git machete github <subcommand>
+where :bash:`<subcommand>` is one of: :bash:`anno-prs`, :bash:`create-pr`, :bash:`retarget-pr`.
+
+Creates, checks out and manages GitHub PRs while keeping them reflected in branch definition file.
+
+To allow GitHub API access for private repositories (and also to correctly identify the current user, even in case of public repositories),
+a GitHub API token with :bash:`repo` scope is required, see :bash:`https://github.com/settings/tokens`. This will be resolved from the first of:
+    1. :bash:`GITHUB_TOKEN` env var,
+    2. current auth token from the :bash:`gh` GitHub CLI,
+    3. current auth token from the :bash:`hub` GitHub CLI.
+
+:bash:`anno-prs`:
+
+  Annotates the branches based on their corresponding GitHub PR numbers and authors.
+  Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
+  Equivalent to :bash:`git machete anno --sync-github-prs`.
+
+:bash:`create-pr [--draft]`:
+
+  Creates a PR for the current branch, using the upstream (parent) branch as the PR base.
+  Once the PR is successfully created, annotates the current branch with the new PR's number.
+
+  If :bash:`.git/info/description` file is present, its contents is used as PR description.
+  If :bash:`.git/info/milestone` file is present, its contents (a single number - milestone id) is used as milestone.
```

Comment:
```suggestion
  If :bash:`.git/info/description` file is present, its contents are used as PR description.
  If :bash:`.git/info/milestone` file is present, its contents (a single number - milestone id) are used as milestone.
```

Reply:
Also grammar nazi: pls use `&mdash;` (https://virtuslab.github.io/tips/#english/grammar-tips-2 point 4) instead of hyphen where applicable

Reply:
sphinx equivalent of that is `---`